usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -577,3 +586,91 @@ def fetch_naics_hierarchy(naics: str) -> dict:
         pass
     results = {"toptier_code": toptier_code, "midtier_code": midtier_code, "base_code": base_code}
     return results
+
+
+def fetch_account_details_award(award_id: int) -> dict:
+    queryset = (
+        FinancialAccountsByAwards.objects.filter(award_id=award_id)
+        .values("disaster_emergency_fund")
+        .annotate(
+            obligated_amount=Sum("obligations_delivered_orders_unpaid_total_cpe"),
+            gross_outlay_amount=Sum("gross_outlay_amount_by_award_cpe"),
+        )
+    )
+    outlay_by_code = []
+    obligation_by_code = []
+    total_outlay = 0
+    total_obligations = 0
+    for row in queryset:
+        total_outlay += row["gross_outlay_amount"] if row["gross_outlay_amount"] is not None else 0
+        total_obligations += row["obligated_amount"] if row["obligated_amount"] is not None else 0
+        if row["disaster_emergency_fund"] is not None:
+            outlay_by_code.append({"code": row["disaster_emergency_fund"], "amount": row["gross_outlay_amount"]})
+            obligation_by_code.append({"code": row["disaster_emergency_fund"], "amount": row["obligated_amount"]})
+
+    results = {
+        "total_account_outlay": total_outlay,
+        "total_account_obligation": total_obligations,
+        "account_outlays": outlay_by_code,
+        "account_obligations": obligation_by_code,
+    }
+    return results
+
+
+def fetch_account_details_idv(award_id: int) -> dict:
+
+    child_award_sql = """
+    select
+        ac.id as award_id
+    from
+        parent_award pap
+        inner join awards ap on ap.id = pap.award_id
+        inner join awards ac on ac.fpds_parent_agency_id = ap.fpds_agency_id and ac.parent_award_piid = ap.piid
```

Comment:
I _think_ we need to exclude child IDVs here too like we do for grandchild awards (`ac.type not like 'IDV%'`).  Most of the time we don't seem to care about values in the IDV as much as we do their non-IDV children.

Reply:
Do we know why we would exclude them? It seems to me that if a child IDV had a specific DEF code or obligated amount unique to it and not its child awards we would want to include it.

Reply:
We need to confirm this with someone smarter than me, but I believe IDVs don't usually have these values or, if they do, they're not supposed to.  Either way, I think we're supposed to ignore most dollar figures on IDVs.  Kristine or Ross would probably be our best bet.

Reply:
The api/v2/awards/ response should only be data on that specific IDV record.

There is another endpoint (perhaps it's api/v2/awards/idvs/amounts/) to provide the aggregated amounts for child and grandchild IDV orders

Reply:
yup, tony is correct, i completely misread the ticket, the IDV amounts should be under a different endpoint

Reply:
The question still applies to the other endpoint, I suppose.